examples/julia-machine-learning/.saturn/saturn.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
It doesn't look like this example edits the `templates.json` files (either hosted or enterprise), but I think it should edit both, right?